modified kill_process method to call SIGKILL only on non-windows

Craig-ling opened the pull request.

jrwdunham commented on old/lib/parser.py:
```diff
@@ -370,14 +371,21 @@ def target():
-                pass
+        
+        # Obtain PID of Operation System.
+        osPID = os.getpid()
```
I believe this is going to get the python process that is currently running. This will result in the program self-terminating, which is not what we want. Was there a reason that we did not continue to use `process.pid`?

Craig-ling replied:
I don't believe we discussed whether or not to use `process.pid` for Windows. This is my bad. I took your code suggestion on Slack too literally.
`if platform.system() is not 'Windows':
     pid = process.pid
     pids = [pid]
     pids.extend(self.get_process_children(pid))
     for pid in pids:
           try:
                os.kill(pid, signal.SIGKILL)
            except OSError:
                 pass
        else:
              pid = process.pid
              pids = [pid]
              pids.extend(self.get_process_children(pid))
              for pid in pids:
                   try:
                       os.kill(pid, signal.SIGTERM)
                   except OSError:
                       pass`
Is probably what we actually want. We should wrap the logic executed in the conditional in a function and pass the OS specific signal function as an argument.
(Edit: the white space in the code formatting isn't there. Anyway, it's a repeat of the pid, for loop, and try block for Windows with signal.SIGTERM instead of SIGKILL).

jrwdunham reviewed (commented):
I am unsure if the `os.getpid()` call is appropriate here. I am testing this.

jrwdunham committed:
Modify kill_process method to call SIGTERM on Windows

- Also add configlocal.ini for when we want to run an OLD locally using
  SQLite and not config from env vars, e.g., in DativeTop (on Windows.)
